packages/core/src/components/ChoiceList/Choice.scss

```diff
@@ -117,7 +117,8 @@ Style guide: components.choice
 }
 
 .ds-c-choice--inverse:focus + label::before {
-  box-shadow: 0 0 0 2px $color-background-inverse, 0 0 2px 4px $color-focus-inverse;
+  box-shadow: 0 0 0 2px $color-background-inverse,
+    0 0 2px 4px $color-focus-inverse;
```

Comment:
I have Prettier set up on save on Atom, so this changed the formatting a bit. I can back this out if need be.

Reply:
I'm cool with if the linter is cool with it, which it seems to be 👍

Reply:
Also kinda interested in adding Prettier to design system